docs/zh-tw/translation-guide.md

```diff
@@ -129,11 +129,11 @@ For example, consider the [JavaScript](/en-US/docs/Web/JavaScript) guide, which
 
 | 原文 | 翻譯 |
 | --- | --- |
-| Try It | 試試看 |
```

Comment:
used in [`EmbedInteractiveExample`](https://github.com/mdn/yari/blob/d89ab8028f93c8ae13b1316a1fc2ee252d91eb4c/kumascript/macros/EmbedInteractiveExample.ejs#L54-L55). So remove this from here (for we would not need to translate this)